docs/admin/pool_operations.md

```diff
@@ -40,64 +38,50 @@ The maximum length of a pool label is 127 characters.
-     space and not total space, the absolute size of a pool
-     created with `--size=percentage%` will be impacted by other
-     concurrent pool create operations. The command output will
-     always list the total capacities in addition to the
-     requested percentage.
-
-3. The `--scm-size` parameter (and optionally `--nvme-size`) can
-   be used to specify the SCM capacity (and optionally the NVMe
-   capacity) _per storage engine_ in **Bytes**.
-   The minimum SCM size is 16 MiB per **target**, so for a storage
-   engine with 16 targets the minimum is `--scm-size=256MiB`.
-   The NVMe size can be zero. If it is non-zero then the minimum
-   NVMe size is 1 GiB per **target**, so for a storage engine
-   with 16 targets the minimum non-zero NVMe size is
-   `--nvme-size=16GiB`.
-   To derive the total pool capacity, these per-engine capacities
-   have to be multiplied by the number of participating engines.
+A pool’s size is determined by two factors: the number of storage engines participating
+in the allocation, and the amount of capacity allocated from each storage tier.
+The capacity can be specified either per engine or as a total across all participating engines.
+DAOS allocates the same amount of storage on each participating engine. If any of the selected
+engines lack sufficient free space to meet the requested allocation, the pool creation will fail.
+
+If neither the --nranks nor --ranks option is specified, the pool will span all storage engines
+in the DAOS system by default. To restrict the pool to a subset of engines, these options can be
+used to either specify the desired number of engines (--nranks) or provide an explicit list of
+engine ranks (--ranks) to be used for the pool.
+
+DAOS provides multiple ways to define the capacity of a pool, offering flexibility depending on
+whether you want to specify an absolute size, use available capacity percentages, or manually set tier-specific values:
+
+- **Set total size with tier ratio (`--size` + `--tier-ratio`)**
+  - Use `--size` (`-z`) to define the **total pool capacity** in **bytes**.
+  - Use `--tier-ratio` (`-t`) to control the split between **SCM** and **NVMe**.
+    - Format: `scm,nvme` (as percentages).
+    - Default: `6,94` → 6% SCM, 94% NVMe.
+    - Examples:
+      - `-z=100TB -t=6,94` → 6 TB SCM, 94 TB NVMe.
+      - `-z=10TB -t=100,0` → SCM-only pool.
+  - Good for: specifying total pool size while controlling SCM/NVMe balance.
+
+- **Use percentage of free space (`--size` as %)**
+  - Define pool size as a **percentage** of available free space.
+  - `--tier-ratio` is **ignored**.
+  - Example: `--size=100%` allocates all currently free space on both tiers.
+  - Notes:
+    - Cannot be used to create SCM-only pools (unless there's no NVMe).
+    - Uses the **minimum free space** across all engines.
+    - Pool size can vary depending on system state.
+    - Command output shows **actual allocated sizes**.
```

Comment:
do we want another line indicating that the pool size will be dependent on the minimum per-rank size available across all ranks and size will be limited based on the lowest common denominator?